bridgenode/server.go

```diff
@@ -5,15 +5,83 @@ import (
 	"fmt"
 	"math"
 	"net"
+	"os"
+	"time"
 
+	"github.com/btcsuite/btcd/chaincfg"
 	"github.com/mit-dci/utreexo/util"
 	"github.com/syndtr/goleveldb/leveldb"
+	"github.com/syndtr/goleveldb/leveldb/opt"
 )
 
+func ArchiveServer(param chaincfg.Params, dataDir string, sig chan bool) error {
+
+	// Channel to alert the tell the main loop it's ok to exit
+	haltRequest := make(chan bool, 1)
+
+	// Channel for ServeBlock() to wait
+	haltAccept := make(chan bool, 1)
+
+	// Handle user interruptions
+	go stopServer(sig, haltRequest, haltAccept)
+
+	_, err := os.Stat(dataDir)
+	if os.IsNotExist(err) {
+		return fmt.Errorf("%s not found, can't serve blocks\n", dataDir)
+	}
+
+	// TODO ****** server shouldn't need levelDB access, fix this
+	ttlpath := "utree/" + param.Name + "ttldb"
+	// Open leveldb
+	o := opt.Options{CompactionTableSizeMultiplier: 8}
```

Comment:
if the value 8 is arbitrarily chosen, maybe it would make sense to point that out

Reply:
True... but we need to get rid of levelDB there next; better to get rid of the whole thing!